docs/langref/src/models.md

```diff
@@ -1,7 +1,8 @@
-## Arrays / Model
+## Arrays and Models
 
-The type array is using square brackets for example `[int]` is an array of `int`. In the runtime, they are
-basically used as models for the `for` expression.
+Arrays are created by wrapping `[` and `]` around the type of the array elements.
+
+In the runtime, arrays are used as models in the `for` expression.
```

Comment:
This is an attempt to introduce the model term in this "Array and Models" section, but it's a little ... poor.

Perhaps what we should have is a paragraph that introduces arrays, and a second paragraph that explains what models are (a term that only exists in our Rust/C++ API) and how properties holding arrays become models when exposed to native code.

```suggestion
Array literals as well as properties holding arrays act as models in`for` expressions.
```